docs/accounts.md

```diff
@@ -0,0 +1,21 @@
+# Blacklight Login Scenarios 
+
+## Unauthenticated User from main menu
+
+## Library Account - From the User Menu Bar
+    * with Netid - devise user type "cas"
```

Comment:
click on Your account' -> 'Library account' will give the user two options: 
1. 'Login with netID'
2. Log in with Alma Account (affiliates)